lib/mbta.js

```diff
@@ -2,6 +2,66 @@
 
 // Code here.
 
+//Dev Notes:
+//Create a function that calculates the number of stops in between stations on the MBTA.
+//Write additional functions as needed.
+
+
+//Not gonna lie, I'm feeling pretty lost on this one.
+
+
+//First, we will define each of the lines on the MBTA as an object.
```

Comment:
You've correctly created some objects that contain information about each stop, but because you've stored them inside an array it makes it difficult to access them properly. For instance, if I wanted to access the `Kendall` stop inside your `redline` array, I would have to know that it existed at position 2 `redline[2]`. A better way to represent these stops would be the way they do it in the solution branch.